src/paralleltimertree.cpp

```diff
@@ -422,16 +425,31 @@ bool ParallelTimerTree::printTimers(double minFraction, const std::map<std::stri
                table.addElement("Other", 1, stats.level[i]-1);
             }
 
-            table.addElement(stats.countSum[i]/nProcessesInPrint);
-            table.addElement(stats.timeSum[i]/nProcessesInPrint);
+            if(nProcessesInPrint>0)
+               table.addElement(stats.countSum[i]/nProcessesInPrint);
+            else
+               table.addElement(0.0);
+            
+            if(nProcessesInPrint>0)
+               table.addElement(stats.timeSum[i]/nProcessesInPrint);
+            else
+               table.addElement(0.0);
+            
             table.addElement(100.0 * stats.timeParentFraction[i]);
-            double imbTime = stats.timeMax[i].val - stats.timeSum[i]/nProcessesInPrint;            
-            if(nProcessesInPrint>1)
+            
+            if(nProcessesInPrint>0 && stats.timeMax[i].val != 0.0) {
+               double imbTime = stats.timeMax[i].val - stats.timeSum[i]/nProcessesInPrint;
                table.addElement(100 * imbTime / stats.timeMax[i].val  * ( nProcessesInPrint /(nProcessesInPrint - 1 )));
+            }
+            else{
+               table.addElement(0.0);
+            }
+
+            if(nProcessesInPrint>0)
```

Comment:
I have tried to have spaces in the expressions for readability reasons:  (nProcessesInPrint > 0)